### DISCUSSION TITLE
二要素認証、Passkeyの対応

### DISCUSSION THREAD

**Discussion by kenta-hi** [topotal/waroom-roadmap#65] [Feature Request]
## Why

セキュリティ上の観点から二要素認証、Passkey等に対応する

## What

二要素認証、Passkeyを実装する

**Comment by sawa-zen** [comment]
パスキー（Passkey）を用いたログイン機能をリリースしました！この新機能により、さらに安全かつ便利にWaroomをご利用いただけます。詳細は以下のページをご覧ください。
https://docs.waroom.com/4ce8573843ec4aea80c1ee6dbc5b720d
